charts/sonatype-nexus/templates/secret.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "nexus.name" . }}-secret
+  name: {{ toYaml .Values.secret.metadata.name }}
```

Comment:
This would need changed in the places that use the secret as well. I recommend adding something to the helpers.tpl and ensuring that all references that use the secret use the function you create in the helpers.tpl